docs/reference/commandline/ps.md

```diff
@@ -51,7 +51,7 @@ The currently supported filters are:
 * exited (int - the code of exited containers. Only useful with `--all`)
 * status (created|restarting|running|paused|exited)
 * ancestor (`<image-name>[:<tag>]`,  `<image id>` or `<image@digest>`) - filters containers that were created from the given image or a descendant.
-* isolation (default|hyperv)   (Windows daemon only)
+* isolation (default|process|hyperv)   (Windows daemon only)
```

Comment:
I'm having a bit of a hard time understanding the use case, it feels like `--filter isolation=I-dont-remember-what-I-set-as-default-but-lets-filter-on-it-because-I-can`